CZ implementations may be nonsymmetric, only calibrated locus orders must be used. (#140)
iqm-finland/qiskit-on-iqm: CHANGELOG.rst

Changelog
=========

Version 15.6
============

> * Qiskit Target now contains CZ with the directions are calibrated on the hardwared. `#140 <https://github.com/iqm-finland/qiskit-on-iqm/pull/140>`_

Reviewer: ```suggestion
* CZ gates now appear in the transpilation target only in the direction they are calibrated. `#140 <https://github.com/iqm-finland/qiskit-on-iqm/pull/140>`_
```
